CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # [vNext]
-
+* Fix: GH28: Define Steps produces incorrect regex for step definition
```

Comment:
This should be something like:
`Fix: Define Steps produces incorrect cucumber expression for step definition (#28)`